protocol-units/bridge/contracts/test/NativeBridge.t.sol

```diff
@@ -111,10 +154,10 @@ contract NativeBridgeTest is Test {
         console.log("Testing correct values");
         nativeBridge.completeBridgeTransfer(bridgeTransferId, _originator, _recipient, _amount, _nonce);
 
-        bytes32 _bridgeTransferId =
-            nativeBridge.noncesToIncomingBridgeTransferIds(_nonce);
+        uint256 nonce =
```

Comment:
See rate limit tests from #917 .

Reply:
I believe now addressed in https://github.com/movementlabsxyz/movement/pull/952

Reply:
See comment below.